xds/src/main/java/io/grpc/xds/internal/security/certprovider/FileWatcherCertificateProvider.java

```diff
@@ -45,6 +46,9 @@ final class FileWatcherCertificateProvider extends CertificateProvider implement
   private static final Logger logger =
       Logger.getLogger(FileWatcherCertificateProvider.class.getName());
 
+  @VisibleForTesting
+  static boolean enableSpiffe = GrpcUtil.getFlag("GRPC_EXPERIMENTAL_SPIFFE_TRUST_BUNDLE_MAP",
```

Comment:
Only have this flag here or in the ProviderProvider. Read the flag from the other class in places that you need it. (e.g., `if (FileWatcherCertificateProviderProvider.enableSpiffe) {`)